### DISCUSSION TITLE
Sobre as rotas que estão sendo criadas.

### DISCUSSION THREAD

**Discussion by ivannaprata** [EasyRoomCompany/Guardian#65] [Q&A]
@kevinDsousa me fala um pouco sobre as próximas rotas que irão ser criadas por você, para que eu possa ir checando a relação correta com os dados do banco. 

Hoje vou testar o arquivo em csv que você gerou para que eu possa testar em futuras análises.

**Comment by kevinDsousa** [comment]
Boa noite @ivannaprata Estou criando aqui as rotas, basicamente funciona como o que chamamos de CRUD (Atualização, Deleção, Adicionar e Puxar) os dados

neste momento estou separando as rotas para organização a fim de deixar mais visualmente entendivél

exemplo de rota
```js
const getUsers = (request, response) => {
  pool.query('SELECT * FROM users ORDER BY id ASC', (error, results) => {
    if (error) {
      throw error;
    }
    response.status(200).json(results.rows);
  });
```

com este codigo somos capazes de fazer uma requisição SQL no banco de dados para retorno de todos usuários cadastrados em ordem ascendente.

O arquivo CSV criado foi em base no banco de dados, acredito que a equipe de Analistas poderão verificar.
